index.html

```diff
@@ -0,0 +1,229 @@
+                <div class="about-parragraf-2">
+                    <p>
+                        Please contact us per email for any further questions about <br> 
+                        Microverse Gaming Fest 2023! <br>
+                        <a href="mailto:jdxelric@gmail.com">microversegamingfest2023@example.com</a>
+                    </p>
+                </div>
+            </div>
+        </section>
+        <section class="logo-section">
+            <div class="logo-main">
+                <div class="logo-title">
+                    <h2 id="logo-title">
+                        Microverse Gaming Fest <br>
+                        Virtual Event 2023 Logo
+                    </h2>
+                    <p>
+                        The logo of Microverse Gaming Fest was made
+                        while building this project xd
+                    </p>
+                </div>
+                <div class="logo-img">
+                    <img src="img/MicroverseGamingFestLogo.png" width="95%" alt="MicroverseGF2023Logo">
+                </div>
+            </div>
+        </section>
+        <section class="past_events-section">
+            <div class="past_events-main">
+                <div class="past_events-title">
+                    <h2 id="past_events-title">
+                        See the past Microverse Gaming Fests
+                    </h2>
+                    <p>
+                        Take a look at the last two Microverse Gaming Fest wich took place
+                        in Saturn, 2022 and in the Earth, 2019.
+                    </p>
+                </div>
+                <div class="past_events-imgs">
+                    <a href="img/logo2k22.png" target="_blank">
+                        <img src="img/logo2k22.png" width="85%" alt="MicroverseGF2022">
+                    </a>
+                    <br>
+                    <a href="img/logo2k19.png" target="_blank">
+                        <img src="img/logo2k19.png" width="85%" alt="MicroverseGF2019">
+                    </a>
+                </div>
+            </div>
+        </section>
+        <section class="partners-section" id="partners-section">
+            <div class="partners-main">
+                <div class="partners-title">
+                    <h2 id="partners-title">
+                        Partners
+                    </h2>
+                </div>
+                <div>
+                    <img src="img/partners_img.png" width="100%" alt="Partners">
+                </div>
+            </div>
+        </section>
+        <footer class="footer-section">
+            <div class="footer-main">
+                <div>
+                    <img src="img/footer_img_desktop.JPG" width="100%" alt="FooterLogo">
+                </div>
+            </div>
+        </footer>
+    </main> 
+    <script src="js/app.js"></script>
+    <!-- <script src="node_modules/bootstrap/dist/js/bootstrap.min.js"></script> -->
+</body>
```

Comment:
- [ ] After the speakers rendering, there's a `more` button needed to implement. Though the functionality is not compulsory, the button is needed 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50990925/233255840-372a182b-4331-4602-9a82-71564a6ef78e.png)

## Design:
![image](https://user-images.githubusercontent.com/50990925/233255879-37fd1a36-db89-44f7-b051-5f27bc433b04.png)